src/Containers/BroadcastContainer.js

```diff
@@ -1,9 +1,8 @@
-import React, { useState, useEffect, useRef, useContext, createContext } from 'react'
+import React, { useState, useEffect, useRef, createContext } from 'react'
 import ChatContainer from './ChatContainer';
 import ParticipantsContainer from './ParticipantsContainer';
 import socketIOClient from "socket.io-client";
 
-const peerConnections = {};
```

Comment:
this may need to be added back? or do we have a peerconnection container in place somewhere else? or is this even it?

Reply:
The peer connections are stored in state:
`const [peers, setPeers] = useState(new Map());`
The temporary "participants" array will also later be removed and replaced with functionality from this actual list of peers